packages/test/test-end-to-end-tests/src/test/gc/gcInactiveNodes.spec.ts

```diff
@@ -21,40 +21,68 @@ import {
 } from "@fluidframework/test-utils";
 import { describeNoCompat, ITestDataObject, itExpects, TestDataObjectType } from "@fluidframework/test-version-utils";
 
+/**
+ * @param settings - feature flags to set
+ * @returns a config provider that returns feature flag information
+ */
+ export const mockConfigProvider = ((settings: Record<string, ConfigTypes>): IConfigProviderBase => {
+    return {
+        getRawConfig: (name: string): ConfigTypes => settings[name],
+    };
+});
+
 /**
  * Validates this scenario: When a GC node (data store or attachment blob) becomes inactive, i.e, it has been
  * unreferenced for a certain amount of time, using the node results in an error telemetry.
  */
-describeNoCompat("GC inactive nodes tests", (getTestObjectProvider) => {
```

Comment:
This file is WIP - I will not be replacing the existing tests, but rather copying them first or updating them to run in 2 modes.